mos-platform/common/lib/main.S

```diff
@@ -8,7 +8,7 @@
 __main:
     TSX
```

Comment:
I need help with this one.  My meager assembly language skills are failing me. If it make __save_s a routine called from main(), how do I get the correct stack entry to store in __exit_s?  If I just take the existing stack value I think I end up back in __main, rather than the caller for __main?

Reply:
Ah, you're right; I hadn't thought this through completely; it'd need to be something that could be injected directly inline, which is more complex than what you already have.

No action required.